Convert django-log-request-id-feedstock to v1 feedstock

rxm7706 opened the pull request:
This PR converts django-log-request-id-feedstock to a v1 recipe and switch the conda build tool to rattler-build.
It has been automatically generated with [feedrattler v0.3.14.dev26+g9fab0e2b0](https://github.com/hadim/feedrattler).

Changes:
- [x] 📝 Converted `meta.yaml` to `recipe.yaml`
- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] 🔧 Updated `conda-forge.yml` to use `rattler-build` and `pixi` (optional)
- [x] 🔢 Bumped the build number
- [x] 🐍 Applied temporary fixes for `python_min` and `python_version`
- [x] 🔄 Rerender the feedstock with conda-smithy
- [ ] Ensured the license file is being packaged.

devin-ai-integration[bot] committed:
Convert to v1 feedstock and use pixi as conda install tool

devin-ai-integration[bot] committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.10.04.03.00.01

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

devin-ai-integration[bot] committed:
Apply recipe.yaml formatting updates and fix conda-forge.yml

- Add yaml-language-server schema comment
- Quote version string in context section
- Add pip_check: true to python imports section
- Remove redundant test item with pip check script
- Change conda-forge.yml inspection from update-grayskull to update-all

rxm7706 commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.04.03.00.0
